app/views/current_optional_subjects/_optional_subjects_list.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="mdc-deprecated-list">
```

Comment:
can we use `app/views/subjects/_subjects_list.html.erb` instead?
i think it's exactly the same

Reply:
Agreed, removed.